aws_marketplace/README.md

```diff
@@ -8,11 +8,13 @@ To know more about algorithms and model packages from AWS Marketplace, see [docu
 
 #### Create algorithms/model packages for listing in AWS Marketplace for machine learning.
 
-This example notebook shows you how to package a model-package/algorithm for listing in AWS Marketplace for machine learning.
+These example notebooks show you how to package a model or algorithm for listing in AWS Marketplace for machine learning.
 
-- [Creating Algorithm and Model Package - Listing on AWS Marketplace](creating_marketplace_products) provides a detailed walkthrough on how to package a scikit learn algorithm to create SageMaker Algorithm and SageMaker Model Package entities that can be used with the enhanced SageMaker Train/Transform/Hosting/Tuning APIs and listed on AWS Marketplace.
+- [Creating Marketplace Products](creating_marketplace_products)
+        - [Creating a Model Package - Listing on AWS Marketplace](creating_marketplace_products/models) provides a detailed walkthrough on how to package a pre-trained model as a SageMaker Model Package that can be listed on AWS Marketplace.
```

Comment:
This nesting isn't working. Preview the markdown file and ensure the indented bullets are rendering correctly.